lib/node/nodes/data-observatory-multiple-measures.js

```diff
@@ -54,46 +54,35 @@ var DataObservatoryMultipleMeasures = Node.create(TYPE, PARAMS, { cache: true, l
 
 module.exports = DataObservatoryMultipleMeasures;
 
+DataObservatoryMultipleMeasures.prototype._composeObsMetaParams = function() {
+    var params = this.numerators.map(function (numerator, index) {
+        return {
+            numer_id: numerator,
+            denom_id: (this.denominators && this.denominators[index]) ?
+                this.denominators[index] : null,
+            normalization: this.normalizations[index],
+            geom_id: (this.geom_ids && this.geom_ids[index]) ?
+                this.geom_ids[index] : null,
+            numer_timespan: (this.numerator_timespans && this.numerator_timespans[index]) ?
+                this.numerator_timespans[index] : null
+        };
+    }.bind(this));
+
+    return JSON.stringify(params);
```

Comment:
Hum there is no array [in the template](https://github.com/CartoDB/camshaft/pull/317/files/5ec93727d9a7b7d2ff97383b06b1ab5a7b588220#diff-684e239f32416fca588420dbea53cce6R127)